client/components/popover/index.jsx

```diff
@@ -129,13 +135,18 @@ class Popover extends Component {
 			return null;
 		}
 
-		if ( ! isVisible || isVisible === prevProps.isVisible ) {
+		if ( ! isVisible ) {
 			return null;
 		}
 
-		this.debug( 'Update position after render completes' );
-
-		setTimeout( () => this.setPosition(), 0 );
+		if ( ! this.isUpdatingPosition ) {
+			this.debug( 'requesting to update position after render completes' );
+			requestAnimationFrame( () => {
```

Comment:
What's the reasoning for this?

Reply:
The reasoning is in the message for 38e779cfebf6dd42bb30d8be7127951b0c531ae2: 

```
Popover: update position even when own props stay the same
Previously, the Popover didn't update its position when its children changed. This would lead to a mis-positioned Popover when the height of its children was changed.

"This change makes sure that the Popover recalculates its position even when its props are the same. This makes sure that it is correctly positioned when only its children change. This change uses `this.isUpdatingPosition` to avoid infinite updating loops.```

Reply:
Might be worth including an inline comment on the code.

Reply:
👍 Good call!